{"status":"ERROR","subtests":[{"name":"# AUDIT TASK RUNNER STARTED.","status":"PASS"},{"name":"> [ref-distance-error]","status":"PASS"},{"name":"X new PannerNode(c, {refDistance: -1}) threw \"TypeError\" instead of RangeError.","status":"FAIL"},{"name":"X new PannerNode(c, {refDistance: 0}) incorrectly threw TypeError: \"Function expected\".","status":"FAIL"},{"name":"X new PannerNode(c, {refDistance: 5e-324}) incorrectly threw TypeError: \"Function expected\".","status":"FAIL"}]}